src/interface/api-outbound.yaml

```diff
@@ -334,6 +334,58 @@ paths:
           $ref: '#/components/responses/501'
         '503':
           $ref: '#/components/responses/503'
+  /consentRequests:
```

Comment:
Why is this on the outbound interface? I think I'm missing something here... Are we reusing the same request but making it syncronous?

Reply:
Yes this outbound synchronous endpoint is for e2e testing purpose

Reply:
I don't love that we are reusing the resource names, but that's a fight for another day...